src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -394,7 +394,7 @@ public static class TextCatalog
         /// <param name="catalog">The text-related transform's catalog.</param>
         /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
         /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
-        /// <param name="hashBits">Number of bits to hash into. Must be between 1 and 30, inclusive.</param>
+        /// <param name="numberOfHashBits">Number of bits to hash into. Must be between 1 and 30, inclusive.</param>
```

Comment:
>numberOfHashBits [](start = 25, length = 16)

Each change in this file will lead to a conflict to my PR...

Reply:
Let's see who finishes first :)